src/app/ApplicationTemplate.Windows/Properties/Properties/win10-x86.pubxml

```diff
@@ -6,7 +6,7 @@ https://go.microsoft.com/fwlink/?LinkID=208121.
   <PropertyGroup>
     <PublishProtocol>FileSystem</PublishProtocol>
     <Platform>x86</Platform>
-    <RuntimeIdentifier>win10-x86</RuntimeIdentifier>
```

Comment:
Should we rename the specific files for this too?

Reply:
Good question.

@LProulx86 can you check this

Reply:
Fix. remove win10 for win